### ISSUE TITLE
[REQUEST] Make OB Smoother & Logging Options

### ISSUE THREAD

**Issue by Unkn9wn1** [openbullet/openbullet#473]
When i get many hits [1000+] OB get hangs, crashes or even exits automatic...
also it use many resources {CPU,RAM} and makes whole windows slower
specially when using OB for information gathering...
and there is many problems with logs

i think this options could make OB better :

\- option to stop logging hits at all
it's good to be in config's options, and in OB's Settings both

\- option to stop logging hits in Runner!
If 'Logging Hits' is enabled in settings and config, this will be needed

\- option to keep hits logs in db and showing them in 'Hits DB'
It's really bad that last ver. of OB can't keep hit's logs
i know it could make DB very big, but it's needed sometimes

\- option to limit OB to keep only last {-} of 'hits', 'customs' and 'to checks' in runner.
after hiding them from 'Runner', just forgot their logs too
showing all of them and cache all logs will just make lot of problems

\- option to stop whole 'Runner Bots Display Mode'
when thread is a lot, it makes OB laggy

\- limit showing captured data in 'Runner' details
it cause no big problems but it's better to see only first 100 chars of captured data and also showing linebreaks as "\n" in captured data, if captured data was too much, user can copy it and paste on notepad to see all of it
Look at this!
![image](https://user-images.githubusercontent.com/41063376/72676643-dd4dc680-3aa8-11ea-81df-d1bb1c02aae7.png)
i can't even what was the data/creds

**Comment by openbullet** [comment]
The real question here is why are you capturing so much data, dump it to a file instead. The entire source code of a website is not something you'd want to store in a database...

**Comment by Unkn9wn1** [comment]
it was only an stupid example, don't focus on it
the problem is that there is no limits at all
huge data, line breaks, thousands hits...

**Comment by openbullet** [comment]
Is this just a "display" issue or do you want a hard limit on the amount of data that can be captured? Should we just truncate the displayed text while maintaining the long string in the database?

**Comment by Unkn9wn1** [comment]
yes, the last one (- limit showing captured data in 'Runner' details) is just a "display" issue

**Comment by Unkn9wn1** [comment]
but :
- option to stop logging hits
- option to keep hit's log in db, and showing them in 'Hits DB'

is all about logging
sometimes we don't need hit's log at all, it could use ram and cpu, and make DB bigger (if OB saves it in DB) we need to disable hits log, like bot log that is disabled by default

sometimes we need hit's log, but after we close runner bot, we lose our hits logs
we need to have our hit's log in "Hits DB"
